_includes/v21.2/misc/storage-classes.md

```diff
@@ -0,0 +1,13 @@
+Use the parameter to set one of the following [storage classes](https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutObject.html#AmazonS3-PutObject-request-header-StorageClass):    
+
+- `STANDARD`
```

Comment:
Are we worried that this could become stale as amazon changes their allowed options? would it make sense to just link to their docs instead of duplicating it?